protocol-units/settlement/mcr/client/src/eth_client.rs

```diff
@@ -252,6 +275,45 @@ where
 	}
 }
 
+pub struct AnvilAddressEntry {
+	pub address: String,
+	pub private_key: String,
+}
+
+/// Read the Anvil config file keys and return all address/private key.
+pub fn read_anvil_json_file_addresses<P: AsRef<Path>>(
```

Comment:
@l-monninger did not document the separation of concerns when adding the *-setup infrastructure crates, but as I understand it, this setup functionality belongs precisely there, not in the client crate. The integration tests would then consume the configuration created by the e2e setup.

Reply:
To be further addressed in #105.